src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -440,7 +440,7 @@ private void LookupExtensionMethodsInSingleBinder(ExtensionMethodScope scope, Lo
             bool resultWithoutSuffixIsViable = IsSingleViableAttributeType(result, out symbolWithoutSuffix);
 
             // Generic types are not allowed.
```

Comment:
>are [](start = 29, length = 3)

nit: `// Generic types *were* not allowed.` (same comment applies below)

Reply:
Done